RookieAND/20230509/72412. 순위 검색/순위 검색.js

```diff
@@ -0,0 +1,112 @@
+function solution(info, querys) {
+  // 배열 자체를 처리하지 않고, 나머지 요소를 key로 변환하여 "hashMap" 을 만들어야 했다.
+  const hashData = new Map();
+  info.forEach((value) => {
+    const data = value.split(' ');
+    const score = Number(data.pop());
+
+    let key = data.join('');
+    hashData.set(
+      key,
+      hashData.has(key) ? [...hashData.get(key), score] : [score],
+    );
+  });
+
+  // Map을 순회하면서 각각의 요소들을 모두 오름차순 한다.
+  for (let [key, value] of hashData) {
+    hashData.set(
+      key,
+      value.sort((a, b) => a - b),
+    );
+  }
```

Comment:
```suggestion
for (let v of map.values()) {
    v.sort((a, b) => a - b);
  }
```

민규님이 알려주신 부분인데, 요렇게 간결하게 소트할 수 있다고 합니다 ㅎㅎ

Reply:
좋은 접근법이네요. 참고하도록 하겠습니다